mongo/src/main/java/com/redhat/lightblue/mongo/config/MongoDBResolver.java

```diff
@@ -55,6 +58,18 @@ public MongoConfiguration getConfiguration(MongoDataStore store) {
         }
         return null;
     }
+    
+    @Override
+    public List<MongoConfiguration> getConfigurations() {
+    	
+    	List<MongoConfiguration> mongoConfigurations = new ArrayList<>();
+        
+    	for (MongoConfiguration cfg : datasources.values()) {
+    		mongoConfigurations.add(cfg);
+        }
+    	
+        return Collections.unmodifiableList(mongoConfigurations);
```

Comment:
I think it's okay to avoid copying the list here and just `return Collections.unmodifiableList(datasources.values())`.